src/randoop/FailureSet.java

```diff
@@ -5,17 +5,27 @@
 import java.util.List;
 import java.util.Set;
 
+import randoop.operation.MethodCall;
+import randoop.sequence.ExecutableSequence;
+import randoop.sequence.MutableSequence;
+import randoop.sequence.MutableVariable;
+import randoop.sequence.Statement;
+import randoop.sequence.Variable;
+
 public class FailureSet {
   
   private Set<Failure> failures = new LinkedHashSet<Failure>();
   
   public static class Failure {
-    public final StatementKind st;
+    
+    public final Statement st;
```

Comment:
This isn't your fault because the documentation was missing before, but can you ensure that when you edit code it always has documentation for each field?
Maybe you needn't do this here because it is just a result of the rename refactoring.  I would have found the pull request easier to understand if you could break it into parts.  At least doing the rename first and then all the rest would help, because that would eliminate a bunch of lines from the diff.  (I hope it wouldn't be too hard to do all those refactorings on an unmodified version of the code.)